### PR TITLE
docs(v2): Add Hasura Docs site to showcase page

### PR DESCRIPTION
**Pull request by surendran82**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

https://deploy-preview-7301--docusaurus-2.netlify.app/showcase/

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -778,6 +778,14 @@ const Users: User[] = [
     source: 'https://github.com/Hashnode/support',
     tags: ['opensource'],
   },
+  {
+    title: 'Hasura',
+    description: 'The fastest way to create a GraphQL API from your data with authorization, real-time subscriptions, and more for free.',
+    preview: require('./showcase/hasura.png'),
+    website: 'https://hasura.io/docs/',
+    source: 'https://github.com/hasura/graphql-engine/tree/master/docs',
+    tags: ['opensource', 'product', 'large'],
```
Comment (slorber) [inline website/src/data/users.tsx:787]:
```suggestion
    tags: ['favorite', 'opensource', 'product', 'large'],
```

Thanks

As suggested to Nicolas we can add it to favorite sites as it's a great one and have a few cool features to showcase

**Review by slorber — CHANGES_REQUESTED** [review]
Great site!

Note we need the Facebook/Meta CLA to be signed or we can't merge

File: website/src/data/users.tsx (hunk 1 above)
Comment (surendran82) [reply website/src/data/users.tsx]: I have signed the CLA, Can you please approve my PR

Comment (slorber) [reply website/src/data/users.tsx]: we have to wait for the CLA bot to confirm, it is usually fast but sometimes take longer

**Review by Josh-Cena — APPROVED** [review]
These GraphQL sites are all amazing...

**Pull request merged**